Tests/LegacyUnit/Service/RegistrationManagerTest.php

```diff
@@ -6513,7 +6523,10 @@ public function createRegistrationCallsSeminarRegistrationCreatedHook()
 
         $uid = $subject->getRegistration()->getUid();
 
-        \Tx_Oelib_Db::delete('tx_seminars_attendances', 'uid = ' . $uid);
+        /** @var ConnectionPool $connectionPool */
+        $connectionPool = GeneralUtility::makeInstance(ConnectionPool::class);
+        $connection = $connectionPool->getConnectionForTable('tx_seminars_attendances');
```

Comment:
We the `getConnectionForTable` calls multiple time in this test case. Please let's have a private method `getConnectionForTable` that encapsulate the `makeInstance` and `getConnectionForTable` calls in order to reduce this duplication.